docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.9'
+services:
+    plugin_server:
+        ports:
+            - ${WEB_LISTEN_PORT_BACKEND:-2222}:${WEB_OUTPUT_PORT_BACKEND:-2222}
+        container_name: plugin_server
+        build:
+            context: server
+            target: base
+            dockerfile: prod.Dockerfile
+        command: node cli.js
+        restart: unless-stopped
+        networks:
+            - default
```

Comment:
Can't this mount a directory that you want to work in? It's weird that you'd need to copy new code into the container.

Reply:
Yes, I can make that change. Just wanted extra security disallowing access to any mount point.